StoreKitUnitTests/TrialOrIntroPriceEligibilityCheckerSK2Tests.swift

```diff
@@ -131,4 +133,122 @@ class TrialOrIntroPriceEligibilityCheckerSK2Tests: StoreKitConfigTestCase {
             expect(receivedEligibility.status) == expected[product]
         }
     }
+
+    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    func testCheckEligibilityForProductIsEligible() async throws {
```

Comment:
I'd name this something like 
```suggestion
    func testCheckEligibilityForProductIsEligibleForEligibleSubscription() async throws {
```

Reply:
which I realize is a long and terrible name anyway, my thinking is that if you see
`testCheckEligibilityForProductIsEligibleForEligibleSubscription failed`, you know exactly which part failed